prometheus/prometheus.spec

```diff
@@ -0,0 +1,201 @@
+
+################################################################################
+
+%define service_user   prometheus
+%define service_group  prometheus
+
+################################################################################
+
+Summary:          Monitoring system and time series database
+Name:             prometheus
+Version:          2.2.1
+Release:          0%{?dist}
+Group:            Applications/Databases
+License:          ASL 2.0
+URL:              https://prometheus.io
+
+Source0:          https://github.com/%{name}/%{name}/archive/v%{version}.tar.gz
+Source1:          %{name}.service
+Source2:          %{name}.init
+Source3:          %{name}.sysconfig
+
+BuildRoot:        %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+
+BuildRequires:    golang >= 1.9
+
+%if 0%{?rhel} >= 7
+Requires:         systemd
+
+Requires(pre):    shadow-utils
+Requires(post):   systemd
+Requires(preun):  systemd
+Requires(postun): systemd
+%else
+Requires:         initscripts kaosv >= 2.15
+
+Requires(pre):    shadow-utils
+Requires(post):   %{__chkconfig}
+Requires(preun):  %{__chkconfig} %{__service}
+Requires(postun): %{__service}
+%endif
+
+Conflicts:       prometheus < 2.0
+
+Provides:        %{name} = %{version}-%{release}
+
+################################################################################
+
+%description
+
+Prometheus is a systems and service monitoring system. It collects metrics from
+configured targets at given intervals, evaluates rule expressions, displays the
+results, and can trigger alerts if some condition is observed to be true.
+
+################################################################################
+
+%prep
+%setup -qn %{name}-%{version}
+
+%build
+export GOPATH=$(pwd)
+go get github.com/%{name}/%{name}/cmd/...
+
+%{__make} %{?_smp_mflags} build
+
+%install
+rm -rf %{buildroot}
+
+install -dm 0755 %{buildroot}%{_bindir}
+install -dm 0755 %{buildroot}%{_sysconfdir}/%{name}
+install -dm 0755 %{buildroot}%{_sysconfdir}/sysconfig
+install -dm 0755 %{buildroot}%{_datarootdir}/%{name}/consoles
+install -dm 0755 %{buildroot}%{_datarootdir}/%{name}/console_libraries
+install -dm 0755 %{buildroot}%{_sharedstatedir}/%{name}
+
+%if 0%{?rhel} >= 7
+    install -dm 0755 %{buildroot}%{_unitdir}
+%else
+    install -dm 0755 %{buildroot}%{_initrddir}
+%endif
+
+install -pm 755 prometheus %{buildroot}%{_bindir}/%{name}
+install -pm 755 promtool %{buildroot}%{_bindir}/promtool
+
+for dir in consoles console_libraries; do
+  for file in ${dir}/*; do
+    install -pm 0644 ${file} %{buildroot}%{_datarootdir}/%{name}/${file}
+  done
+done
+
+install -pm 0644 documentation/examples/%{name}.yml %{buildroot}%{_sysconfdir}/%{name}/%{name}.yml
+install -pm 0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/sysconfig/%{name}
+
+%if 0%{?rhel} >= 7
+    install -pm 0644 %{SOURCE1} %{buildroot}%{_unitdir}/%{name}.service
+%else
+    install -pm 0755 %{SOURCE2} %{buildroot}%{_initrddir}/%{name}
+%endif
+
+%check
```

Comment:
Empty `%check` section.

Reply:
I will add check for this to `perfecto`.

Reply:
Fixed (17ae9ae54df2d89f4bee20225412d11859903292)